modules/integration/src/test/scala/scala/cli/integration/InstallHomeTests.scala

```diff
@@ -0,0 +1,98 @@
+package scala.cli.integration
+
+import com.eed3si9n.expecty.Expecty.expect
+
+import scala.util.Properties
+
+class InstallHomeTests extends munit.FunSuite {
+
+  val firstVersion            = "0.0.1"
+  val secondVersion           = "0.0.2"
+  val dummyScalaCliFirstName  = "DummyScalaCli-1.scala"
+  val dummyScalaCliSecondName = "DummyScalaCli-2.scala"
+  val dummyScalaCliBinName    = "scala-cli-dummy-test"
+  val testInputs = TestInputs(
+    Seq(
+      os.rel / dummyScalaCliFirstName ->
+        s"""
+           |object DummyScalaCli extends App {
+           |  println(\"$firstVersion\")
+           |}""".stripMargin,
+      os.rel / dummyScalaCliSecondName ->
+        s"""
+           |object DummyScalaCli extends App {
+           |  println(\"$secondVersion\")
+           |}""".stripMargin
+    )
+  )
+  val binDirPath = os.home / ".scala" / "scala-cli"
```

Comment:
We shouldn't use the local `os.home` in the tests here (this is going to bork any Scala CLI installed there).